### PR TITLE
[TIMOB-10368] iOS: Expose "color" and "hintTextColor" to Ti.UI.SearchBar

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-10368

### CODE REVIEW

File: apidoc/Titanium/UI/SearchBar.yml [hunk 1]
```diff
@@ -165,11 +165,28 @@ properties:
     since: "5.4.0"
     platforms: [iphone,ipad]
 
+  - name: color
+    summary: Color of the text in this text field, as a color name or hex triplet.
+    description: |
+        For information about color values, see the "Colors" section of <Titanium.UI>.
+    type: String
+    platforms: [iphone, ipad]
```
Comment (jquick-axway) [inline apidoc/Titanium/UI/SearchBar.yml]: Android already supports this, just wasn't documented. Would you mind adding "android" to the supported platform list please?

File: apidoc/Titanium/UI/SearchBar.yml [hunk 2]
```diff
@@ -165,11 +165,28 @@ properties:
...
+    since: "7.1.0"
+
   - name: hintText
     summary: Text to show when the search bar field is not focused.
     type: String
     default: On iOS, "Search"; on Android, no hint text.
 
+  - name: hintTextColor
+    summary: Hint text color to display when the field is empty.
+    platforms: [iphone, ipad]
```
Comment (jquick-axway) [inline apidoc/Titanium/UI/SearchBar.yml]: Android already supports this property too.

File: tests/Resources/ti.ui.searchbar.addontest.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	afterEach(function () {
+		if (win) {
+			win.close();
+		}
+		win = null;
+	});
+	
+	// TODO: Expose on Android / Windows as well
+	// We have in in Ti.UI.Android.SearchView for Android, but need more parity here
+	it.ios('.hintTextColor', function () {
```
Comment (jquick-axway) [inline tests/Resources/ti.ui.searchbar.addontest.js]: Android can run this unit test too.
Change to `it.windowsMissing`?

File: tests/Resources/ti.ui.searchbar.addontest.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+		should(searchBar.getHintTextColor).be.a.Function;
+		should(searchBar.hintTextColor).eql('red');
+		should(searchBar.getHintTextColor()).eql('red');
+		searchBar.hintTextColor = 'blue';
+		should(searchBar.hintTextColor).eql('blue');
+		should(searchBar.getHintTextColor()).eql('blue');
+	});
+	
+	// TODO: Expose on Android / Windows as well
+	it.ios('.color', function () {
```
Comment (jquick-axway) [inline tests/Resources/ti.ui.searchbar.addontest.js]: Android can run this unit test.
Change to `it.windowsMissing`?

**Review by jquick-axway — CHANGES_REQUESTED** [review]

**Review by vijaysingh-axway — APPROVED** [review]
CR passed.

**Comment by hansemannn** [comment]
@jquick-axway Can you update your review?

**Review by jquick-axway — APPROVED** [review]
CR: Pass

**Comment by ewieberappc** [comment]
FR Passed. Tested with 7_1_X branch PR

**Pull request merged**